### ISSUE TITLE
Data analysis

### ISSUE THREAD

**Issue by JonaJJSJ-crypto** [JonaJJSJ-crypto/Proyecto-de-Tesis#54]

**Comment by JonaJJSJ-crypto** [comment]
Here we are studying the how to filter our dataset at different levels here listed:

1. No filter
2. Trigger selection
3. Trigger Object selection
4. Hq quality tracks and jets selection
5. Signal selection
All the proposed histograms can be found in the next [repository](https://github.com/JonaJJSJ-crypto/Data3/tree/main/analysis3) inside the directory with the same name

**Comment by JonaJJSJ-crypto** [comment]
About the Quantities: Here will be listed the distributions that are considered in every cut and the motivation to include it.

1. Number of primary vertices
2. Invariant mass from Two Dijets
3. Invariant mas from electron+Two dijets (LWelectron decay)
4. Number of secondary vertices in the event (without merging)
5. Displaced from secondary to primary vertex
6. DeltaR between electrons
7. DeltaR between electrons and jets
8. DeltaR between jets
9. Electron pt
10. Jet pt(Most energetic, second most energetic)

**Comment by JonaJJSJ-crypto** [comment]
@caredg 
Here is the full macro for analysis: https://github.com/JonaJJSJ-crypto/Data3/blob/main/analysis3/EventLoopAnalysisTrigger.cxx

**Comment by caredg** [comment]
> @caredg In this Macro you can see how I am making the different cuts. I'm still thinking in a smart way to plot all the histograms at once

I do not see any link here

**Comment by caredg** [comment]
@JonaJJSJ-crypto, cuando dices "el momento corregido de MC esta ligeramente desplazado del momento de los Datos", significa que para los datos no estás tomando el pT corregido?  Además, cómo se ve el gráfico con el pT sin corregir?

**Comment by JonaJJSJ-crypto** [comment]
@caredg no, me refiero a que los datos y mc están tomando el momento corregido pero parece que en mc o en datos hay este desplazamiento extraño

**Comment by JonaJJSJ-crypto** [comment]
@caredg  come new distributions for secondary vertices and new cuts are already in the [Repository](https://github.com/JonaJJSJ-crypto/Data3/tree/main/analysis5/TriggerFilter/DRC)

**Comment by JonaJJSJ-crypto** [comment]
@caredg [Repository](https://github.com/JonaJJSJ-crypto/Data3/tree/main/analysis5/Histograms/PNG) is better organized new variables would be ready soon

**Comment by JonaJJSJ-crypto** [comment]
@caredg The repository is updated

**Comment by JonaJJSJ-crypto** [comment]
@caredg El [repositorio](https://github.com/JonaJJSJ-crypto/Data3/tree/main/analysis9/Histograms) esta actualizado.

**Comment by JonaJJSJ-crypto** [comment]
@caredg El [repositorio](https://github.com/JonaJJSJ-crypto/Data3/tree/main/analysis11/Histograms) con incertidumbres esta actualizado.
